dockers/docker-fpm-frr/base_image_files/TS

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# read SONiC immutable variables
+[ -f /etc/sonic/sonic-environment ] && . /etc/sonic/sonic-environment
+
+PLATFORM=${PLATFORM:-`sonic-cfggen -H -v DEVICE_METADATA.localhost.platform`}
+
+# Parse the device specific asic conf file, if it exists
+ASIC_CONF=/usr/share/sonic/device/$PLATFORM/asic.conf
+[ -f $ASIC_CONF ] && . $ASIC_CONF
+
+if [[ ($NUM_ASIC -gt 1) ]]; then
+    asic=0
+    while [ $asic -lt $NUM_ASIC ]
```

Comment:
@judyjoseph I think we should not run on backend asic. It will impact Internal BGP Session route-map FROM_BGP_INTERNAL_PEER_V4 and FROM_BGP_INTERNAL_PEER_V6

Reply:
TSA/B/C works only on the routemaps which are attached as outbound on BGP neighbors, these route maps will have name starting with "TO_*". (https://github.com/Azure/sonic-buildimage/blob/master/dockers/docker-fpm-frr/TSA)

I did for all the ASICs here as anyways the device is going into maintenance mode with Taffic shift, and should be fine. With TSA, the Front-end BGP instances will not send routes to its external and internal ASIC neighbors. The back-end BGP instances will not send routes to its internal ASIC neighbors.

I can fine tune this to apply only on the EXTERNAL outbound route maps in the fornt-end ASIC.

Reply:
Updated to run TSA/B/C on the front end ASIC BGP instances. 
In addition skip processing of internal routemaps in the front-end ASIC's